clang/lib/Sema/TreeTransform.h

```diff
@@ -11499,6 +11509,28 @@ TreeTransform<Derived>::TransformBuiltinBitCastExpr(BuiltinBitCastExpr *BCE) {
                                                 Sub.get(), BCE->getEndLoc());
 }
 
+template <typename Derived>
```

Comment:
I'm not sure how to decide when to do so, but many of the transforms here check if the sub-transforms actually DID anything, or if AlwaysRebuild is set.

Reply:
Done